api/v1alpha1/dataprotectionapplication_types.go

```diff
@@ -452,6 +451,8 @@ type NodeAgentConfig struct {
 	// How long to wait for resource processes which are not covered by other specific timeout parameters. Default is 10 minutes.
 	// +optional
 	ResourceTimeout *metav1.Duration `json:"resourceTimeout,omitempty"`
+	// TODO: remove restic from below uploaderType enum?
+
 	// The type of uploader to transfer the data of pod volumes, the supported values are 'restic' or 'kopia'
 	// +kubebuilder:validation:Enum=restic;kopia
```

Comment:
Shouldn't we be removing the uploader type option as "restic" for install ? https://github.com/vmware-tanzu/velero/releases/tag/v1.17.0